filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -1418,6 +1418,8 @@ void OpenGLDriver::createSwapChainR(Handle<HwSwapChain> sch, void* nativeWindow,
 
     GLSwapChain* sc = handle_cast<GLSwapChain*>(sch);
     sc->swapChain = mPlatform.createSwapChain(nativeWindow, flags);
+    PANIC_POSTCONDITION("createSwapChain(%p, 0x%lx) failed. See logs for details.",
```

Comment:
should these be `ASSERT_POSTCONDITION` on `sc-swapChain`?

Reply:
yes! thanks!